src/zabapgit_popups.prog.abap

```diff
@@ -76,18 +76,46 @@ CLASS lcl_popups DEFINITION FINAL.
         RAISING   lcx_exception
                   lcx_cancel,
       popup_to_select_transports
-        RETURNING VALUE(rt_trkorr) TYPE trwbo_request_headers.
+        RETURNING VALUE(rt_trkorr) TYPE trwbo_request_headers,
+      popup_select_obj_overwrite
+        IMPORTING it_list TYPE lif_defs=>ty_results_tt
+        RETURNING VALUE(rt_list) TYPE lif_defs=>ty_results_tt
+        RAISING   lcx_exception.
 
   PRIVATE SECTION.
     TYPES: ty_sval_tt TYPE STANDARD TABLE OF sval WITH DEFAULT KEY.
 
-    CLASS-METHODS: add_field
-      IMPORTING iv_tabname    TYPE sval-tabname
-                iv_fieldname  TYPE sval-fieldname
-                iv_fieldtext  TYPE sval-fieldtext
-                iv_value      TYPE clike DEFAULT ''
-                iv_field_attr TYPE sval-field_attr DEFAULT ''
-      CHANGING  ct_fields     TYPE ty_sval_tt.
+    TYPES BEGIN OF t_popup_select_list.
+      TYPES selected TYPE flag.
+      INCLUDE TYPE lif_defs=>ty_result.
+    TYPES END OF t_popup_select_list.
```

Comment:
Hi, can you change this part to

```abap
TYPES: BEGIN OF t_popup_select_list,
         selected TYPE flag.
    INCLUDE TYPE lif_defs=>ty_result.
TYPES END OF t_popup_select_list.
```

it will fix the abaplint error